docs/source/installation.rst

```diff
@@ -11,18 +11,10 @@ Chemprop can either be installed from PyPI via pip_, from source (i.e., directly
 .. _conda: https://docs.conda.io/en/latest/conda.html
 .. _miniconda: https://docs.conda.io/en/latest/miniconda.html
 
-.. note:: 
-    We also plan to make chemprop installable using an ``environment.yml`` file with ``conda`` before the release of v2.0.0.
-
-Start by setting up your virtual environment. We assume you are using ``conda`` or ``miniconda``, but you may adapt these steps use any virtual environment manager you like:
-
-.. code-block::
-
-    conda create -n chemprop python=3.11
-    conda activate chemprop
+We assume you are using ``anaconda`` or ``miniconda`` to set up your local environment, but you may adapt these steps use any virtual environment manager you like (e.g. ``mamba``):
 
 .. note:: 
-    *CPU-only installation:* If you do not have a GPU, install a CPU-only version of PyTorch by running the following command before installing Chemprop:
+    *CPU-only installation:* For the following options, if you do not have a GPU, install a CPU-only version of PyTorch by running the following command before installing Chemprop:
```

Comment:
Could you explicitly say which options, i.e. 1-4?

Reply:
just options 1-3, right? Not sure how this would work with Docker. 

I'm not sure I understand why we need this instruction to begin with...will pip/conda ever try to install a GPU version when no GPU is present? I've only had the opposite happen (having a CPU version installed on a GPU machine when the GPU isn't detected for some reason).

Reply:
Yeah I think 1-3, Docker is CPU already

pip/conda **usually** gets the right version... I don't think this is needed, though some offline conversation with others is what led it to be here.